SEED-platform/seed
# Add public geojson endpoint

#### Any background context you want to provide?

#### What's this PR do?
- Adds a public geojosn endpoint that leverages some functionality from the tax lot property export code
- Limits fields returned to those set in org sharing and available gis data
- Adds an org setting for public_geojson_enabled

#### How should this be manually tested?
With properties that have geo data like lat/long,
1. Test with endpoints disabled. Go to org settings and unselect either "enable public endpoints" or "enable public geojson". Go to /api/v3/public/organizations/{org_id}/cycles/{cycle_id}/geo.json. The endpoints should return an error message

2. Test with endpoints enabled. Go to /api/v3/public/organizations/{org_id}/cycles/{cycle_id}/geo.json. Copy the json response and paste into [geojson.io](geojson.io). Map should show your seed properties

3. Add query_param "?taxlots=true" to return only taxlots


#### What are the relevant tickets?
#4635

#### Screenshots (if appropriate)
![Screenshot 2024-05-03 at 11 22 19 AM](https://github.com/SEED-platform/seed/assets/58446472/10a8d0ab-d650-43d6-b4bb-f747d58781b1)
![Screenshot 2024-05-14 at 2 08 16 PM](https://github.com/SEED-platform/seed/assets/58446472/6c36632d-1947-4436-b7b2-21583d56e30f)
